test/controllers/home_page_controller_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Community.HomePageControllerTest do
+  use Community.ConnCase
+
+  test "GET / shows approved posts", %{conn: conn} do
+    approved = build(:job, %{
+      city: "Dvegas",
+      company: "big company name",
+      title: "approved",
+    }) |> approve |> create
+    not_approved = create(:job, %{title: "SPAM"})
+
+    conn = get conn, "/"
+    refute html_response(conn, 200) =~ not_approved.title
+
```

Comment:
I think this newline should be removed and placed above the refute. Probably just a mistype :)